edx_sga/tests.py

```diff
@@ -152,6 +186,9 @@ def test_max_score_integer(self):
     @mock.patch('edx_sga.sga.render_template')
     @mock.patch('edx_sga.sga.Fragment')
     def test_student_view(self, Fragment, render_template):
+        """
+        Test student view renders correctly.
+        """
```

Comment:
It won't let me comment there, but on line 195 render_template.assert_called_once is called out by pylint as a useless statement, which it is.  It should be `render_template.assert_called_once()` as it is currently not actually calling the function.